packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -171,6 +173,8 @@ const DATA_MAP: AuthDataMap = {
     }
 }
 
+export const ninetyDaysInSeconds = 90 * 24 * 60 * 60
```

Comment:
it feels weird to have a exported variable like this. Would you consider naming it something like a `DEFAULT_SLAS_REFRESH_TOKEN_TTL`? (all caps since it's a constant)

Reply:
Addressed.